bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml

```
@@ -546,7 +546,7 @@ spec:
                               format: int64
                               type: integer
                             garbage-collection-frequency:
-                              description: How long to wait by default before backups can be garbage collected. (default 720h0m0s)
+                              description: How often garbage collection is run for expired backups. (default once per hour)
```

More clarity like so? almost :)
```suggestion
                              description: How often garbage collection checks for expired backups. (default once per hour)
```

---

Thanks @kaovilai, I will patch it